src/Features/Core/Portable/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.GlobalSuppressMessageFixAllCodeAction.cs

```diff
@@ -81,12 +94,19 @@ private static async Task<Solution> CreateChangedSolutionAsync(AbstractSuppressi
                 {
                     var oldProject = kvp.Key;
                     var currentProject = currentSolution.GetProject(oldProject.Id);
-                    var diagnosticsBySymbol = await CreateDiagnosticsBySymbolAsync(fixer, oldProject, kvp.Value, cancellationToken).ConfigureAwait(false);
-                    if (diagnosticsBySymbol.Any())
+                    var compilation = await currentProject.GetCompilationAsync(cancellationToken).ConfigureAwait(false);
+                    var suppressMessageAttribute = compilation.SuppressMessageAttributeType();
+
+                    if (suppressMessageAttribute != null)
```

Comment:
Seems like this can never be null.

Reply:
We will not create a GlobalSuppressMessageCodeAction if the attribute is not defined, so likely can never create a GlobalSuppressMessageFixAllCodeAction. I am fine if you want to still retain the null check, in case you want to guard against it being created from some API context.

Reply:
why is tht true?

Reply:
this is the static functoin that makes the action and produces the doc.  So ths is the point at which we're actuallys checking if we have the suppression attribute.  i think we need this.

Reply:
yeah, i'm having trouble proving this is true.  i'm still looking.

Reply:
yeah, i don't think this is true.  will keep the null checks for now.